NOTICE.txt

```diff
@@ -1114,13 +1114,19 @@ Apache License 2.0
 
 --------------------------------------------------------------------
 Dependency: github.com/elastic/go-sysinfo
-Version: v1.1.0
-Revision: 51d9d1362d77a4792dfb39a7a19f056cdf1b9840
+Version: v1.2.0
+Revision: bd0a84193268134cbe6cdd21b0a5618aee1f5791
 License type (autodetected): Apache-2.0
 ./vendor/github.com/elastic/go-sysinfo/LICENSE.txt:
 --------------------------------------------------------------------
 Apache License 2.0
 
+-------NOTICE.txt-----
+Elastic go-sysinfo
+Copyright 2017-2019 Elasticsearch B.V.
```

Comment:
2020?

Reply:
Good point. It should be updated in elastic/go-sysinfo.